File: 04-tp/2026-09-07-tp3-atelier-de-crash/scripts-casses/05_silencieux.py
# -----------------------------------------------------------------
# 05 -- Exportations francaises de grumes de chene, 2022-2024
#
# Ce script ne plante pas. Il produit un nombre.
# Votre travail : decider si ce nombre est vrai.
# -----------------------------------------------------------------

import pandas as pd

trade = pd.read_csv("data/raw/comtrade_fr_roundwood_clean.csv")

oak = trade[(trade.cmdCode == 440391)
            & (trade.reporterDesc == "France")
            & (trade.flowDesc == "Export")]

total = oak.primaryValue.sum()

print("Exportations francaises de grumes de chene, 2022-2024 :")
print(f"   {total / 1e6:.1f} millions USD")
print(f"   {len(oak)} flux declares")

File: 04-tp/2026-09-07-tp3-atelier-de-crash/scripts-casses/05_silencieux.R
# -----------------------------------------------------------------
# 05 -- Exportations francaises de grumes de chene, 2022-2024
#
# Ce script ne plante pas. Il produit un nombre.
# Votre travail : decider si ce nombre est vrai.
# -----------------------------------------------------------------

trade <- read.csv("data/raw/comtrade_fr_roundwood_clean.csv")

oak <- subset(trade,
              cmdCode      == 440391 &
              reporterDesc == "France" &
              flowDesc     == "Export")

total <- sum(oak$primaryValue)

cat("Exportations francaises de grumes de chene, 2022-2024 :\n")
cat("  ", round(total / 1e6, 1), "millions USD\n")
cat("  ", nrow(oak), "flux declares\n")
